packages/discovery-provider/ddl/migrations/0051_usdc_purchase_access.sql

```diff
@@ -0,0 +1,24 @@
+begin;
+
+  -- disable triggers
+  alter table usdc_purchases disable trigger on_usdc_purchase;
+
+  -- add columns to usdc_purchases
+  alter table usdc_purchases
+  add column if not exists is_streamable boolean default true;
```

Comment:
why not make this an enum too? i imagine we will add album, etc. etc.

Reply:
"album" seems like a `content_type`